src/main/java/epam/task/third/entities/Cone.java

```diff
@@ -0,0 +1,74 @@
+package epam.task.third.entities;
+
+import java.util.Objects;
+
+public class Cone implements Cloneable{
+    private static final Integer DEFAULT_ID = 0;
+    private Point baseCentre;
+    private Point apexPoint;
+    private Double radius;
+    private Integer ID = DEFAULT_ID;
```

Comment:
private int id;
no need to assign a default, pass it in the constructor as a mandatory parameter